bin/version-manager.js

```diff
@@ -140,7 +140,7 @@ function run(files) {
   runner.on('end', viewer.end.bind(viewer))
 
   runner.on('packageResolved', function(pkg, versions) {
-    console.log(pkg + ': ' + versions.length)
+    console.log(pkg + ': ' + versions.join(', '))
```

Comment:
Could we also keep the count for sanity checking? Could come just before the listed versions.

Like `koa-router (45): x.x, x.y`

Reply:
I believe this may list prior to the sampling, which seems to occur in the matrix file. 

https://github.com/newrelic/node-test-utilities/blob/main/lib/versioned/matrix.js#L140-L155

Which I think is fine but for the sampling cases (aws-sdk) we may also want additional logging to display what was sampling down to.

Reply:
> I believe this may list prior to the sampling, which seems to occur in the matrix file.
> 
> https://github.com/newrelic/node-test-utilities/blob/main/lib/versioned/matrix.js#L140-L155
> 
> Which I think is fine but for the sampling cases (aws-sdk) we may also want additional logging to display what was sampling down to.

good catch. I'll dig a little more to provide a closer list of which versions were tested

Reply:
actually, i found an even better place to do this. I'm goin to to tweak all of this